### ISSUE TITLE
Cluttered navigation in osmand.

### ISSUE THREAD

**Issue by ADepic** [osmandapp/Osmand#5649]
When I try tell people about osmand, a common retort is the cluttered navigation (and the slow rendering but that is a separate issue).
![screenshot_osmand _20180629-084800](https://user-images.githubusercontent.com/24532838/42409706-cb0c6a20-81cd-11e8-96a0-3fa2a061a4d3.png)

As you can see, everything is squashed at the top, making navigating quite a bit cluttered.
Many other mapplications used to bottom to place details.

![](https://corp.imgsmail.ru/media/2016/08/31/mapsme-launches-bicycle-navigation2.png)
![screenshot_maps_20180707-111736](https://user-images.githubusercontent.com/24532838/42409892-2a85db6a-81d0-11e8-9d1c-a4a6e1070250.png)

Here you can see navigation from Maps (fork of Maps.Me) where a lot of information is placed on the bottom, making maps.me not cluttered for both landscape mode. 
If you want to access a few settings, all you have to do is press the triangle in the bottom right corner to access traffic, settings and volume.
![screenshot_maps_20180707-111739](https://user-images.githubusercontent.com/24532838/42409810-491ad798-81cf-11e8-8877-943cfefc0e2e.png)

Another good example is Magic earth
![screenshot_magic_earth_20180707-114306](https://user-images.githubusercontent.com/24532838/42410120-36d4795e-81d4-11e8-8699-476cf5e97f73.png)
Here magic earth places speed info and time info in the bottom left and bottom right, respectively. THis gives more space for the top.

To access more stuff, such as overview, search, settings and roadblock options you simply tap on the screen to get that info.
![screenshot_magic_earth_20180707-114309](https://user-images.githubusercontent.com/24532838/42410125-43acd4a0-81d4-11e8-9ff1-452e9933a7ee.png)

Google maps does this too:
![google maps navigation](https://proxy.duckduckgo.com/iu/?u=http%3A%2F%2Fcdn.ndtv.com%2Ftech%2Fimages%2Fgoogle_maps_app_screenshot_routes_app_store.jpg&f=1)

As you can see by the image on the left all the time details are at the botttom

Osmand needs to be more space efficient for navigation. Probably most devices that navigate with osmand are mobile phones, and not tablets. But even for tablets this wouldn't be space inefficient.

**Comment by TheAdventurer64** [comment]
Yes. I completely agree!

**Comment by ADepic** [comment]
Yeah, well this app will always be really bad, just like all osm trash

**Comment by TheAdventurer64** [comment]
Not everything is OSM trash. There are some decent apps out there like StreetComplete. The Mapbox SDK is up to par with Google Maps.

**Comment by seniorm0ment** [comment]
I agree, I had made a feature request a while back about offering a simpler GUI, and a simpler map style more along the lines of Google Maps.
While I totally understand some people want detail, thats fine. 
But I think a map style clone of google maps or waze is really needed.

I also feel the gui looks a bit outdated. The icons don't look the cleanest either.

**Comment by vshcherb** [comment]
We plan to rearrange top right into bottom widget so it will look like 2-3 elements for navigation, though it will be a bit disbalanced left / right side but it will look more organized. This will be optional

**Comment by seniorm0ment** [comment]
Could it be beneficial to make it so users can just drag elements around where they want?
